freqtrade/freqai/data_kitchen.py

```diff
@@ -244,6 +244,14 @@ def filter_features(
                 f"{self.pair}: dropped {len(unfiltered_df) - len(filtered_df)} training points"
                 f" due to NaNs in populated dataset {len(unfiltered_df)}."
             )
+            if len(unfiltered_df) == 0 and not self.live:
+                raise OperationalException(
```

Comment:
What was current behavior in backtesting? Didn't it simply log the error and move on?

i wonder if this won't this hinder a longer-term backtest (say, all of 2022) with pairs that only listed in April 2022? (that's a valid scenario, happening more often than we think).

Reply:
No, if there is no data, the backtest fails with a zero division error. Backtests only work if the user has the data. Therefore, you cannot backtest a coin on all of 2022  if you only have data from April. You need to start your backtest in probably March 2022 (depends on how much training data the user asks for).

Reply:
Ok, i'll open a seperate issue for this (it's not directly related to this PR).

That's a scenario freqtrade supports, so it should be a scenario freqAI supports as well.